test/edu/ucsb/nceas/metacat/startup/MetacatInitializerIT.java

```diff
@@ -0,0 +1,152 @@
+package edu.ucsb.nceas.metacat.startup;
+
+import java.io.IOException;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+
+import org.dataone.configuration.Settings;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.mockito.Mockito;
+
+import edu.ucsb.nceas.LeanTestUtils;
+import edu.ucsb.nceas.metacat.index.queue.IndexGenerator;
+import edu.ucsb.nceas.metacat.properties.PropertyService;
+import edu.ucsb.nceas.metacat.util.SystemUtil;
+import edu.ucsb.nceas.utilities.PropertyNotFoundException;
+
+/**
+ * The IT test class for the MetacatInitializer class
+ * @author tao
+ *
+ */
+public class MetacatInitializerIT {
+    private ServletContextEvent event = null;
+
+    @Before
+    public void setUp() throws Exception {
+        LeanTestUtils.initializePropertyService(LeanTestUtils.PropertiesMode.LIVE_TEST);
+        event = getMockServletContextEvent();
+    }
+
+    @After
+    public void tearDown() {
+    }
+
+    /**
+     * Test the contextInitialized method
+     * @throws IOException
+     */
+    @Test
+    public void testContextInitialized() throws Exception {
+        //This should be a full-configured Metacat
+        MetacatInitializer metacatInitializer = new MetacatInitializer();
+        metacatInitializer.contextInitialized(event);
+        assertTrue(metacatInitializer.isFullyInitialized() == true);
```

Comment:
don't need the `== true`, i.e.: `assertTrue(metacatInitializer.isFullyInitialized());`